docs/querying/api.md

```diff
@@ -683,6 +683,46 @@ $ curl http://localhost:9090/api/v1/status/flags
 
 *New in v2.2*
 
+###  Runtime Info
+
+The following endpoint returns runtime info of the Prometheus server.
+
+*Note that the exact values returned may change over time*
+
+```
+GET api/v1/status/runtimeinfo
+```
+
+```json
+$ curl http://localhost:9090/api/v1/status/runtimeinfo
+{
+  "status": "success",
+  "data": {
+    "alertmanagers": null,
```

Comment:
It would be nice to add a non-null value.

```
...
    "alertmanagers": [
      {
        "scheme": "http",
        "host": "localhost:9093",
        "path": "/api/v1/alerts"
      }
    ],
...
```